hazelcast/src/main/java/com/hazelcast/sql/impl/client/SqlExecuteMessageTask.java

```diff
@@ -58,7 +58,8 @@ protected Object call() {
 
         SqlServiceImpl sqlService = nodeEngine.getSqlService();
 
-        return sqlService.execute(query, sqlSecurityContext, parameters.queryId);
+        boolean skipUpdates = parameters.isSkipUpdateStatisticsExists && parameters.skipUpdateStatistics;
```

Comment:
Minor: I believe we could improve the name of this variable. `skipUpdates` might be a bit misleading. Maybe we could also mention the `statistics` in the variable name as well